core/templates/dev/head/pages/exploration_editor/StatePropertyService.js

```diff
@@ -125,27 +125,27 @@ oppia.factory('stateInteractionIdService', [
 // interaction. This is a dict mapping customization arg names to dicts of the
 // form {value: customization_arg_value}.
 // TODO(sll): Add validation.
-oppia.factory('stateCustomizationArgsService', [
-  'statePropertyService', function(statePropertyService) {
-    var child = Object.create(statePropertyService);
+oppia.factory('StateCustomizationArgsService', [
+  'StatePropertyService', function(StatePropertyService) {
+    var child = Object.create(StatePropertyService);
     child.setterMethodKey = 'saveInteractionCustomizationArgs';
     return child;
   }
 ]);
 
 // A data service that stores the current interaction hints.
-oppia.factory('stateHintsService', [
-  'statePropertyService', function(statePropertyService) {
-    var child = Object.create(statePropertyService);
+oppia.factory('StateHintsService', [
+  'StatePropertyService', function(StatePropertyService) {
+    var child = Object.create(StatePropertyService);
     child.setterMethodKey = 'saveHints';
     return child;
   }
 ]);
 
 // A data service that stores the current interaction solution.
-oppia.factory('stateSolutionService', [
-  'statePropertyService', function(statePropertyService) {
-    var child = Object.create(statePropertyService);
+oppia.factory('StateSolutionService', [
+  'StatePropertyService', function(StatePropertyService) {
+    var child = Object.create(StatePropertyService);
     child.setterMethodKey = 'saveSolution';
     return child;
   }
```

Comment:
There's a bunch of constants below this line that don't seem relevant to a StatePropertyService. Can we put it somewhere else instead? Perhaps ExplorationEditor.js?